mobile/lib/repositories/album_media.repository.dart

```diff
@@ -14,7 +14,6 @@ class AlbumMediaRepository implements IAlbumMediaRepository {
     final List<AssetPathEntity> assetPathEntities =
         await PhotoManager.getAssetPathList(
       hasAll: true,
-      filterOption: FilterOptionGroup(containsPathModified: true),
```

Comment:
What does this do?

Reply:
Basically it excludes all of the empty album